packages/taskserver/src/cron-tempfile-deletion.ts

```diff
@@ -0,0 +1,46 @@
+import { ProjectInterface } from '@etherealengine/common/src/interfaces/ProjectInterface'
+
+export const getCronJobBody = (project: ProjectInterface, image: string): object => {
+  return {
+    apiVersion: 'cronjob-2',
+    kind: 'CronJob-tempfile-deletion',
```

Comment:
Are the apiVersion and kind fields a) necessary for this body and b) the correct format if so? The project-update cronjob getCronJobBody didn't seem to need those fields, likely because the k8BatchClient call with it is filling them in. But also, I don't recognize either of those field values. apiVersion is usually something like `v1` or `batch/v1beta1`, and kind for CronJobs is usually 'CronJob'. If those are values you filled in to distinguish it, know that they have very specific purposes and accepted values.

Reply:
Not necessarily needed here. I will remove those fields as I can see they are in the k8BatchClient. Thanks.